TESTING.md

```diff
@@ -89,9 +89,24 @@ Automated testing is provided with Jenkins for Continuous Integration. Jenkins e
 Selenium tests are run in headless mode on CI. Locally the same tests will be executed in a real browser. You can activate headless mode by setting the environment variable:
 `export TEST_BROWSER_HEADLESS=1`
 
+Since local Selenium tests are run in a real browser, the dev environment should have a desktop environment and a Google Chrome installed to run the tests.
```

Comment:
I believe Google Chrome isn't explicit? Like I believe Chromium is even supported.

Reply:
Okay, ill update that 👍